frontend/src/widgets/TaskCard/taskcard-styles.js

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/react';
+
+export const taskCard = (theme) => css`
+  margin-left: 14px;
+  margin-right: 18px;
+  margin-bottom: 20px;
```

Comment:
ideally left and right margins shouldn't be handled by the task card, since the padding on the columns would be sufficient